docs/data/base/components/number-field/number-field.md

```diff
@@ -9,27 +9,51 @@ waiAria: https://www.w3.org/WAI/ARIA/apg/patterns/spinbutton/
 
 # Number Field
 
-<p class="description">The Number Field component provides users with a field for number values, with stepper buttons and a scrub area to increment or decrement the value.</p>
+<p class="description">Number Field provides users with a numeric input, with buttons and a scrub area to increment or decrement its value.</p>
 
 {{"component": "modules/components/ComponentLinkHeader.js", "design": false}}
 
 {{"component": "modules/components/ComponentPageTabs.js"}}
 
-## Introduction
+{{"demo": "UnstyledNumberFieldIntroduction", "defaultCodeOpen": false, "bg": "gradient"}}
 
-A number field is a UI element that accepts numeric values from the user. `NumberField` is a customizable replacement for the native HTML `<input type="number">` that solves some usability and visual issues while enhancing its functionality.
+## Installation
 
-{{"demo": "UnstyledNumberFieldIntroduction", "defaultCodeOpen": false, "bg": "gradient"}}
+Base UI components are all available as a single package.
 
-## Component
+<codeblock storageKey="package-manager">
 
-```jsx
+```bash npm
+npm install @base_ui/react
+```
+
+```bash yarn
+yarn add @base_ui/react
+```
+
+```bash pnpm
+pnpm add @base_ui/react
+```
+
+</codeblock>
+
+Once you have the package installed, import the component.
+
+```ts
 import * as NumberField from '@base_ui/react/NumberField';
 ```
 
-### Anatomy
+## Anatomy
+
+NumberField is implemented using a collection of related components:
```

Comment:
If we're referencing the actual component—as we want to emphasise via capitalisation—we should omit the space, since we're not talking about a "number field" in general, we're talking about Base UI's "NumberField".

Reply:
Ok I updated it